packages/circuit-ui/components/Body/Body.docs.mdx

```diff
@@ -0,0 +1,26 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+
+# Body
+
+<Status.Stable />
+
+Body is used to present the core textual content for our users. They are contained within headlines and subheadlines
+and have five different variants `highlight`, `quote`, `success`, `error` and `subtle` to tailor it according to the content we are
+presenting.
+
+<Story id="typography-text--base" />
```

Comment:
These story ids are outdated:
```suggestion
<Story id="typography-body--base" />
```

Please double-check the other component docs as well.